docs/getting-started.md

```diff
@@ -114,6 +114,16 @@ Console.ReadLine();
 host.Stop();
 ```
 
+## Registry
+
+The workflow host keeps workflow definitions in the registry when they are registered.  When starting a workflow, the workflow host creates a new instance for the desired workflow definition.
```

Comment:
The workflow host does not keep the definitions, the registry does, the workflow host accesses the registry.